### PR TITLE
Yarn, versionbump, cleanup, and re-build.

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,17 +23,17 @@
...
+    "@actions/core": "^1.10.1",
+    "execa": "^8.0.1"
   },
   "devDependencies": {
-    "@vercel/ncc": "^0.24.1",
-    "eslint": "^7.4.0",
-    "jest": "^26.4.2"
+    "@vercel/ncc": "^0.38.1",
+    "eslint": "^8.52.0",
+    "jest": "^29.7.0"
```
Comment (JamieSinn) [inline package.json:37]: This should resolve a lot of the issues we have on dependabot.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,18 @@
 {
   "name": "carthage-bootstrap",
-  "version": "1.0.0",
+  "version": "1.1.0",
+  "packageManager": "yarn@3.6.3",
```
Comment (JamieSinn) [inline package.json:4]: Moving this to use Yarn

File: package.json (hunk 1 above)
Comment (chris-hoefgen) [reply package.json:37]: yeah, some big bumps but given how out of date the repo was it makes sense.

**Review by chris-hoefgen — APPROVED** [review]
I think it all makes sense. Is this action only used on the ios-client-sdk repo?

**Pull request merged**